dkube-minimal/kustomization.yaml

```diff
@@ -0,0 +1,121 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+# Cert-Manager
+#- ../common/cert-manager/cert-manager/base
+#- ../common/cert-manager/kubeflow-issuer/base
+# Istio
+- ../common/istio-1-9/istio-crds/base
+- ../common/istio-1-9/istio-namespace/base
+- ../common/istio-1-9/istio-install/base
+# OIDC Authservice
+#- ../common/oidc-authservice/base
+# Dex
+#- ../common/dex/overlays/istio
+# KNative
+#- ../common/knative/knative-serving/base
+#- ../common/knative/knative-eventing/base
+#- ../common/istio-1-9/cluster-local-gateway/base
+# Kubeflow namespace
+- ../common/kubeflow-namespace/base
+# Kubeflow Roles
+- ../common/kubeflow-roles/base
+# Kubeflow Istio Resources
+- ../common/istio-1-9/kubeflow-istio-resources/base
+
+
+# Kubeflow Pipelines
+# - ../apps/pipeline/upstream/env/platform-agnostic-multi-user-legacy
+## Install argo workflow controller and minio separately as pipeline component is not installed(argo controller and minio are installed with pipeline component by default)
+- argo
+- minio
```

Comment:
I am assuming you tried using argo minio directly from Kubeflow-manifests first. What issues you are facing.

If possible, it would be better to customize everything from this kustomization.yaml

Reply:
@deepio-oc Yes, first I tried to install from Kubeflow-manifests but namespace is not specified for these componenets(argo, minio).